ocaml/typing/typeopt.ml

```diff
@@ -16,18 +16,17 @@
 (* Auxiliaries for type-based optimizations, e.g. array kinds *)
 
 open Path
-open Layouts
 open Types
 open Asttypes
 open Typedtree
 open Lambda
 
 type error =
-    Non_value_layout of type_expr * Layout.Violation.t option
-  | Non_value_sort of Sort.t * type_expr
+    Non_value_jkind of type_expr * Jkind.Violation.t option
```

Comment:
As in the `transl` files, my intuition is that this error is properly about layouts.